installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh

```diff
@@ -27,7 +27,7 @@ function inArray() {
 }
 
 lambda_services=("jazz_cognito-authorizer" "jazz_cloud-logs-streamer" "jazz_services-handler" "jazz_events-handler" "jazz_environment-event-handler" "jazz_deployments-event-handler" "jazz_asset-event-handler" "jazz_slack-event-handler")
-nodejs61_service=("jazz_email" "jazz_usermanagement" "jazz_codeq" "jazz_metrics" "jazz_slack-event-handler" "jazz_is-slack-channel-available" "jazz_admin" "jazz_slack-channel" "jazz_deployments-event-handler" "jazz_assets")
+nodejs81_service=("jazz_email" "jazz_usermanagement" "jazz_codeq" "jazz_metrics" "jazz_slack-event-handler" "jazz_is-slack-channel-available" "jazz_admin" "jazz_slack-channel" "jazz_deployments-event-handler" "jazz_assets")
```

Comment:
Why not other lambda services like - `jazz_cognito-authorizer`, `jazz_cloud-logs-streamer` etc.? Shouldn't we upgrade all nodejs based services (of type: api & function)?

Reply:
they are already taken care of in the code

Reply:
Hmm, I don't think so. I think we are just checking if a service is in nodejs81_service list and if so, updating the runtime to 8.X. Now, since `jazz_cognito-authorizer` is not in this list, we are still updating its runtime as `nodejs6.10`. Is that correct?

Reply:
ya i got your point. updating